src/components/Card.js

```diff
@@ -10,19 +10,23 @@ class Card extends Component {
         const { name, description, from } = this.props.kudos
         return (
             <StyledCard>
-                <Row>
+                <Row margin={'0 0 8px'}>
```

Comment:
I would recommend:
```
const CardRow = styled(Row)`
  margin: ...;
`
```
in order to avoid inline styles.

Reply:
I added a margin to KudosContent which is between this two Rows with inline margins. This way I can add one margin property and not playing with props or one-property-components.
:D 
Thanks @kgajowy & @Tuhaj